lib/vhpi/VhpiCbHdl.cpp

```diff
@@ -263,7 +283,7 @@ int VhpiSignalObjHdl::set_signal_value(long value)
             return -1;
         }
     }
-    if (vhpi_put_value(GpiObjHdl::get_handle<vhpiHandleT>(), &m_value, vhpiDeposit)) {
+    if (vhpi_put_value(GpiObjHdl::get_handle<vhpiHandleT>(), &m_value, vhpiDepositPropagate)) {
```

Comment:
Is changing from `vhpiDeposit` to `vhpiDepositPropagate` necessary as part of this change?